### DISCUSSION TITLE
Rosé Pine Midnight community fork

### DISCUSSION THREAD

**Discussion by Courtcircuits** [rose-pine/neovim#188] [Show and tell]
### Is your feature request related to a problem? Please describe.

I really like rose pine on VSCode because it has a warm palette. Strangely, it's not the same palette as the neovim colorscheme.

### Describe the solution you'd like

It would be cool to have a new variation of the neovim colorscheme looking like the "rosé pine" scheme on vscode.

### Describe alternatives you've considered

My first theme was nordic.nvim, but it was too light, then I used rose pine, but it is too cold, so in the end I forked rose-pine.nvim and changed the background color. 

It looks like this : 

![image](https://github.com/rose-pine/neovim/assets/90451752/c5104e12-f011-4ce6-8021-7a29514e8278)

Here is the link to my fork : https://github.com/Courtcircuits/neovim-darker

Basically, I just changed a few colors in the palette. Since it's a minor codebase update impacting strongly the color scheme, I'm not sure, it's relevant to merge it to the main project. Still, if some people like the idea, let's merge it !

**Comment by mvllow** [comment]
Our palette is the same between VSCode and Neovim, however usage may vary. 

If you're talking about the background colour then that could be a discrepancy caused by the terminal emulator, treesitter, tmux, or a variety of other things. It is possible to override the background in Rosé Pine but for more fine-grained control please look at our canary branch as this adds a method of changing the palette, e.g. a darker "base".

**Comment by Courtcircuits** [comment]
My bad, that's because I use Rosé Pine Burnt on VS Code. Sorry for the inconvenience.

**Reply by mvllow** [reply]
Not a problem, appreciate the fork. We have been considering adding a darker variant for some time but haven't quite landed somewhere we like yet. We will leave this discussion up for others to discover :)

**Reply by Courtcircuits** [reply]
Thanks !
